cmake/configure/configure_1_cuda.cmake

```diff
@@ -96,6 +102,10 @@ MACRO(FEATURE_CUDA_FIND_EXTERNAL var)
       SET(${var} FALSE)
     ENDIF()
 
+    # cuSOLVER requires OpenMP
+    FIND_PACKAGE(OpenMP)
+    SET(DEAL_II_LINKER_FLAGS "${DEAL_II_LINKER_FLAGS} ${OpenMP_CXX_FLAGS}")
```

Comment:
Acked - looks good. But I cannot test at the moment. (Living in IKEA boxes and no system with cuda available.)

Reply:
Is the failing `cuda/solver_02` finally passing for you with this fix?

Reply:
No unfortunately :(